sentry/src/main/java/io/sentry/SendFireAndForgetEnvelopeSender.java

```diff
@@ -26,7 +26,7 @@ public SendFireAndForgetEnvelopeSender(
     Objects.requireNonNull(options, "SentryOptions is required");
 
     final String dirPath = sendFireAndForgetDirPath.getDirPath();
-    if (!hasValidPath(dirPath, options.getLogger())) {
+    if (dirPath == null || !hasValidPath(dirPath, options.getLogger())) {
```

Comment:
`hasValidPath` already checks for NPE

Reply:
Same reason as here https://github.com/getsentry/sentry-java/pull/1439#discussion_r628094104